assembl/static2/js/app/containers/authentication.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Translate } from 'react-redux-i18n';
+
+const Authentication = () => (
+  <p><Translate value="authentication.panelTitle" /></p>
```

Comment:
Yes that's a failure of this system, that's why I suggest to use namespaces. I used this system in my last projects and it's not so bad. However, this system is simplest than Assembl v2 ;)